setup.py

```diff
@@ -12,17 +12,12 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
-if __name__ == "__main__":
-
+try:
     with open("README.md", "r") as handle:
         long_description = handle.read()
+except:
+    long_description = "\n".join(short_description[2:])
 
-    # set requirements
-    install_requires = [
-        'numpy>=1.19.5',
-        'mdanalysis>=2.0.0b0',
-        'mdanalysistests>=2.0.0b0',
```

Comment:
MDAnalysisTests aren't used anywhere so I just removed this from `install_requires`

Reply:
Ok, I was going to leave it to use it in the long term, but not currently using it is a good point.

Reply:
Assuming you only need them for the tests I guess you can it in tests_require eventually? MDAnalysisTests is actually quite large (~ 40 MB), so if you can avoid users downloading it they'll appreciate it.